src/apis/Dimensions/index.js

```diff
@@ -44,6 +46,26 @@ export default class Dimensions {
       scale: win.devicePixelRatio || 1,
       width: win.screen.width
     };
+
+    if (canUseDOM) {
+      const change: window.Event = new window.Event(EVENT_CHANGE);
```

Comment:
You could probably just store an array of listener functions within Dimensions and not bother with an event emitter or the DOM. It'd be faster, too.